src/scripts/mongo/change_password.js

```diff
@@ -0,0 +1,19 @@
+function changePassword(email, newPassword, callback) {
+  const mongo = require('mongodb');
+  const bcrypt = require('bcrypt');
+
+  mongo('mongodb://user:pass@mymongoserver.com/my-db', function (db) {
+    const users = db.collection('users');
+
+    bcrypt.hash(newPassword, 10, function (err, hash) {
+      if (err) {
+        callback(err);
```

Comment:
Let's be consistent in our formatting. 
Some places have:
```if (err) return callback(err);```

while others:
```if (err) { return callback(err); }```

and this:
```if (err) {
  callback(err);
} else {
 ...
}
```

I personally prefer the 1st option, but up for discussion.